modules/stanford_bean_types_featured/scss/stanford_bean_types_featured.scss

```diff
@@ -44,9 +44,123 @@ div[class*="featured"] {
   }
 }
 
-.span6 .featured-block {
-  margin-left: 0;
-  margin-right: 0;
+.front {
+  .fullwidth {
+    .paragraphs-item-p-featured {
+      &.has-image {
+        .group-headline-h2 {
+          margin-top: .2em;
+          margin-bottom: 20px;
+        }
+      }
+    }
+  }
+}
+
+.front .fullwidth .paragraphs-item-p-featured {
+  div.group-overlay-text {
+    .field-name-field-p-featured-description {
+      .field-item {
+        margin: 0 28%;
+
+        @media (max-width: 979px) {
+
```

Comment:
Same as comment on previous PR. Wondering if we want a line break after each media query (mixin or otherwise).

Reply:
code climate complained about not having a line break one day, so I just add it from now on. Long story... I'll tell ya all about it tomorrow :P

Reply:
Resolving this based on discussion on Injector 80 PR for cleaning up CC.